src/commonMain/kotlin/teksturepako/pakku/api/platforms/CurseForge.kt

```diff
@@ -121,11 +121,19 @@ object CurseForge : Platform(
     ): List<CfModModel.File> = this
         .filter { file ->
             file.gameVersions.any { it in mcVersions } && file.sortableGameVersions
-                .filter { it.gameVersionTypeId == LOADER_VERSION_TYPE_ID } // Filter to loader only
+                .filter { it.gameVersionTypeId == LOADER_VERSION_TYPE_ID }
                 .takeIf { it.isNotEmpty() }
                 ?.map { it.gameVersionName.lowercase() }?.any {
-                    loaders.any { loader -> loader == it } || it in validLoaders // Check default valid loaders
-                } ?: true // If no loaders found, accept model
+                    it in loaders || it in validLoaders
+                } ?: true
+        }
+
+    private fun List<CfModModel.File>.sortByLoaders(loaders: List<String>) = this.sortedWith { a, b ->
+            val aVersions = a.sortableGameVersions.filter { it.gameVersionTypeId == LOADER_VERSION_TYPE_ID }
+                .map { it.gameVersionName.lowercase() }
+            val bVersions = b.sortableGameVersions.filter { it.gameVersionTypeId == LOADER_VERSION_TYPE_ID }
+                .map { it.gameVersionName.lowercase() }
+            loaders.indexOfFirst { it in aVersions } - loaders.indexOfFirst { it in bVersions }
         }
```

Comment:
What `a` and `b` stand for? It's not clear to me what it does.
Can you name it so it is more clear?

Reply:
Does this preserve the order of the config when there are more than 2 loaders?